Detectors/EMCAL/base/src/Digit.cxx

```diff
@@ -8,30 +8,45 @@
 // granted to it by virtue of its status as an Intergovernmental Organization
 // or submit itself to any jurisdiction.
 
-#include <iostream>
 #include "EMCALBase/Digit.h"
+#include <TMath.h>
+#include <iostream>
 
 using namespace o2::EMCAL;
 
 ClassImp(Digit)
 
-Digit::Digit(Int_t module, Int_t tower, Double_t amplitude, Double_t time):
-FairTimeStamp(time),
-mModule(module),
-mTower(tower),
-mAmplitude(amplitude)
+  Digit::Digit(Int_t tower, Double_t amplitude, Double_t time)
+  : FairTimeStamp(time), mTower(tower), mAmplitude(amplitude)
+{
+}
+
+bool Digit::operator<(const Digit& other) const { return GetTimeStamp() < other.GetTimeStamp(); }
+
+bool Digit::operator>(const Digit& other) const { return GetTimeStamp() > other.GetTimeStamp(); }
+
+bool Digit::CanAdd(const Digit other)
 {
+  return (mTower == other.GetTower() && TMath::Abs(GetTimeStamp() - other.GetTimeStamp()) <= 25);
 }
```

Comment:
same here

Reply:
As a side note, our function name convention says function names should start lowercase.

Reply:
That was probably some leftover from previous implementation. To be fixed. Other classes (detector, geometry) suffer from this as well.